packages/expo-updates/android/src/androidTest/java/expo/modules/updates/launcher/SelectionPolicyFilterAwareTest.java

```diff
@@ -0,0 +1,147 @@
+package expo.modules.updates.launcher;
+
+import android.net.Uri;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
+import expo.modules.updates.UpdatesConfiguration;
+import expo.modules.updates.db.entity.UpdateEntity;
+import expo.modules.updates.manifest.NewManifest;
+
+@RunWith(AndroidJUnit4ClassRunner.class)
+public class SelectionPolicyFilterAwareTest {
+  JSONObject manifestFilters;
+  SelectionPolicyFilterAware selectionPolicy;
+
+  UpdateEntity updateDefault1;
+  UpdateEntity updateDefault2;
+  UpdateEntity updateRollout1;
+  UpdateEntity updateRollout2;
+  UpdateEntity updateNested;
+
+  @Before
+  public void setup() throws JSONException {
+    manifestFilters = new JSONObject("{\"releaseName\": \"rollout\"}");
```

Comment:
```suggestion
    manifestFilters = new JSONObject("{\"branchName\": \"rollout\"}");
```

We are renaming `release` to `branch`

Reply:
thanks, changed (although the nice thing about this implementation is that it's totally agnostic of the actual keys inside of manifestFilters!)